lib/std/zig/number_literal.zig

```diff
@@ -103,7 +101,6 @@ pub fn parseNumberLiteral(bytes: []const u8) Result {
             },
             'e', 'E' => if (base == 10) {
                 float = true;
-                if (base != 10 and base != 16) return .{ .failure = .{ .invalid_float_base = 2 } };
```

Comment:
Be sure to remove this as a possible error.

Reply:
Closing #15461 in favor of this